packages/oft-evm/test/OFT.t.sol

```diff
@@ -51,13 +58,18 @@ contract OFTTest is TestHelperOz5 {
     address public userA = makeAddr("userA");
     address public userB = makeAddr("userB");
     address public userC = makeAddr("userC");
+    address public userD = makeAddr("userD");
     uint256 public initialBalance = 100 ether;
+    uint256 public initialNativeBalance = 1000 ether;
+
+
+    address public constant NATIVE_TOKEN_ADDRESS = address(0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE);
```

Comment:
Add a comment like:

```
// must match NativeOFTAdapter.sol
```

Reply:
I will be removing this in favour of using `address(0)` as [discussed above](https://github.com/LayerZero-Labs/devtools/pull/838#discussion_r1716762372)